client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -59,10 +59,10 @@ object FlightComponents {
     val port: String = if (flight.ActPax > 0) flight.ActPax.toString else "n/a"
     val last: String = flight.LastKnownPax.getOrElse("n/a").toString
     val max: String = if (flight.MaxPax > 0) flight.MaxPax.toString else "n/a"
-
+    val PortTotPax: Int = flight.ActPax + flight.TranPax
```

Comment:
Just realised that this is actually not going to give us the correct number. ActPax is already the total pax on the flight. We take TranPax away from it to give the pax going to PCP.

Do we want to display something like;

"Port: <portPcpPax> = <port> - <tranPax> transfer"

To be more in line with the API line above it?